qiskit/utils/quantum_instance.py

```diff
@@ -136,6 +136,7 @@ def __init__(
         measurement_error_mitigation_shots: Optional[int] = None,
         job_callback: Optional[Callable] = None,
         mit_pattern: Optional[List[List[int]]] = None,
+        max_tries: int = 50,
```

Comment:
Should we call this something more meaningful for a (new) user looking at the parameters here - how about something like `job_retry_limit` or `max_job_retries` - max tries on it own is not very self explanatory.